plaidml2/bridge/keras/__init__.py

```diff
@@ -1547,8 +1557,21 @@ def square(x):
 
 
 @_log_call
-def squeeze(x, axis):
-    return _KerasNode('squeeze', tensor=plaidml_op.squeeze(x.tensor, axis))
+def squeeze(x, axis=None):
+    if axis is None:
+        # define axis
+        axis = []
+        for s in range(len(x._keras_shape)):
+            if x._keras_shape[s] == 1:
+                axis.append(s)
+    if type(axis) is list:
+        x_squeezed = x
+        for i in range(len(axis)):
+            ax = axis[i] - i
+            x_squeezed = _KerasNode('squeeze', tensor=plaidml_op.squeeze(x_squeezed.tensor, ax))
+        return x_squeezed
```

Comment:
For many cases this won't matter, but it would be nice to get multi-axis squeeze moved into the plaidml_op version of squeeze; applying it multiple times could lead to a performance hit. That could be a separate PR though.

Reply:
I was planning on making that a separate PR -- this one's been gnarly enough as-is.